Update send-events.mdx (#6011)
PostHog/posthog.com: contents/docs/getting-started/send-events.mdx

## Event ingestion nuances

> It's a priority for us that events are fully processed and saved as soon as possible. Typically the events will be usable in queries within a couple of minutes.

Reviewer: ```suggestion
It's a priority for us that events are fully processed and saved as soon as possible. Typicall, events will be usable in queries within a few minutes.
```
